src/locales/en.json

```diff
@@ -4,67 +4,67 @@
     "about": "About",
     "spreadsheet": "Spreadsheet",
     "contactus": "Contact Us",
-    "whitepaper": "Whitepaper"
+    "whitepaper": "White Paper"
```

Comment:
👍

Reply:
The wikipedia article is "White paper" so I guess that's the proper way to go.

Reply:
That is my thinking.